src/autolabel/models/openai.py

```diff
@@ -1,17 +1,18 @@
 from functools import cached_property
 from typing import List, Optional
 import logging
-
-from langchain.chat_models import ChatOpenAI
-from langchain.llms import OpenAI
-from langchain.schema import LLMResult, HumanMessage
 import tiktoken
 
 from autolabel.models import BaseModel
 from autolabel.configs import AutolabelConfig
 from autolabel.cache import BaseCache
 from autolabel.schema import RefuelLLMResult
 
+from langchain.chat_models import ChatOpenAI
```

Comment:
lets move this to inside the model so that we don't fail if openai is not installed